### PR TITLE
Test staged application

### PR DESCRIPTION
**Pull request by marcospereira**
Follow up to #73.

### CODE REVIEW

File: scripts/test-stage [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    exit 1
+fi
+
+# Have a valid application secret
+app_secret=`openssl rand -base64 32`
+
+# Start the application in background
+./target/universal/stage/bin/play-java-jpa-example -Dplay.http.secret.key="${app_secret}" &
+
+success=true
```
Comment (mkurz) [inline scripts/test-stage]: What about this line? What does it do?

Comment (marcospereira) [reply scripts/test-stage]: It was a leftover. :-)

**Comment by mkurz** [comment]
Still fails

**Comment by marcospereira** [comment]
@mkurz now it is green.

**Pull request merged**